File: MoviesExample/app/src/main/java/com/novoda/demo/movies/model/Movie.java
package com.novoda.demo.movies.model;

import com.google.gson.annotations.SerializedName;

public class Movie {

    public String id;
    public String title;
    public String poster_path;
    public String backdrop_path;

    @SerializedName("vote_average")
    public double rating;

    @Override
    public String toString() {
        return "Movie{" +
                "title='" + title + '\'' +
                ", rating=" + rating +
                '}';
    }

    public String posterUrl() {
        return "http://image.tmdb.org/t/p/w92" + poster_path;
    }

    public String backDropUrl() {
        return "http://image.tmdb.org/t/p/w300" + poster_path;
    }
}

File: MoviesExample/core/src/main/java/com/novoda/demo/movies/model/Movie.kt
package com.novoda.demo.movies.model

import com.google.gson.annotations.SerializedName

data class Movie(
        val id: String? = null,
        val title: String? = null,
        val poster_path: String? = null,
        val backdrop_path: String? = null,
        @SerializedName("vote_average")
        val rating: Double = 0.toDouble()
) {

    fun posterUrl(): String = "http://image.tmdb.org/t/p/w92" + poster_path!!
    fun backDropUrl(): String = "http://image.tmdb.org/t/p/w300" + poster_path!!
}
